### PR TITLE
Add support for @see in markdown generation

### PR DESCRIPTION
**Pull request by tdely**
Adds options `--see` to include `@see` tags in the generated markdown.

### CODE REVIEW

File: src/PHPDocsMD/Console/PHPDocsMDCommand.php [hunk 1]
```diff
@@ -145,6 +153,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
                         $docs .= '> '.$class->getDescription().PHP_EOL.PHP_EOL;
                 }
 
+                if ($includeSee && $seeArray = $class->getSee()) {
+                    foreach ($seeArray as $see) {
+                        $docs .= 'See ' . $see . '<br />' . PHP_EOL;
```
Comment (bueltge) [inline src/PHPDocsMD/Console/PHPDocsMDCommand.php:171]: a simple `<br>` should right, html5 style.

File: src/PHPDocsMD/DocInfoExtractor.php [hunk 2]
```diff
@@ -171,4 +175,25 @@ private function figureOutParamDeclaration($words, $currentNameSpace)
 
         return false;
     }
+
+    /**
+     * @param $words
+     * @return array|false
```
Comment (bueltge) [inline src/PHPDocsMD/DocInfoExtractor.php:182]: array|bool describe it better, I mean.

File: src/PHPDocsMD/DocInfoExtractor.php [hunk 3]
```diff
@@ -171,4 +175,25 @@ private function figureOutParamDeclaration($words, $currentNameSpace)
...
+     */
+    private function figureOutSeeDeclaration($words)
+    {
+        array_shift($words);
+
+        if (!$words) {
+            return false;
```
Comment (bueltge) [inline src/PHPDocsMD/DocInfoExtractor.php]: `$see = false;` get you the chance to use only one return statement, more readable.

File: src/PHPDocsMD/Console/PHPDocsMDCommand.php (hunk 1 above)
Comment (bueltge) [reply src/PHPDocsMD/Console/PHPDocsMDCommand.php:171]: That's also a topic for the other files in this PR.

Comment (tdely) [reply src/PHPDocsMD/Console/PHPDocsMDCommand.php:171]: I went with `<br />` to match the already present `<hr />` at line 144. But it's all good, I'll change it.

Comment (tdely) [reply src/PHPDocsMD/Console/PHPDocsMDCommand.php:171]: On second thought I'll leave it as is until the maintainer requests differently, at which point would be better to change all instances.

**Pull request merged**